src/lib.rs

```diff
@@ -39,9 +39,9 @@ pub struct Challenge {
 #[derive(Clone, Serialize, Deserialize, Debug)]
 pub struct TeePubKey {
     pub alg: String,
-    #[serde(rename = "k-mod")]
+    #[serde(rename = "n")]
     pub k_mod: String,
-    #[serde(rename = "k-exp")]
+    #[serde(rename = "e")]
```

Comment:
As long as the on wire format complies with the RFC, we should be fine and we can keep more descriptive field names for the structure?

Reply:
@sameo @slp This was my thought as well. As long as it's correct in JSON when (de)serializing, it wouldn't matter much. If it's desired to keep them uniform in the structs themselves (i.e. renaming `k_exp` to `e`), that's fine.

Reply:
@tylerfanelli I personally prefer keeping the structure fields more readable (`k_exp` reads better than `e` in code).

Also, do you think we could tag a `0.3.0` so that we can move back to a JWK compliant wire format?